cumulusci/core/config/universal_config.py

```diff
@@ -2,16 +2,23 @@
 from pathlib import Path
 
 from cumulusci.core.config import BaseTaskFlowConfig
-from cumulusci.core.config.project_config import BaseProjectConfig
+from cumulusci.core.config.project_config import (
+    BaseProjectConfig,
+    ProjectConfigPropertiesMixin,
+)
 from cumulusci.core.utils import merge_config
 from cumulusci.utils.yaml.cumulusci_yml import cci_safe_load
 
 __location__ = os.path.dirname(os.path.realpath(__file__))
 
 
-class UniversalConfig(BaseTaskFlowConfig):
+class UniversalConfig(BaseTaskFlowConfig, ProjectConfigPropertiesMixin):
     """Base class for the global config which contains all configuration not specific to projects"""
 
+    universal_config_obj: dict  # undocumented
```

Comment:
universal_config_obj is used on BaseProjectConfig, not on UniversalConfig

Reply:
Moved in ee235fe01

Reply:
I see you moved it, but it's still the wrong type. It's a UniversalConfig instance, not a dict